plugins/com.google.cloud.tools.eclipse.projectselector/src/com/google/cloud/tools/eclipse/projectselector/ProjectRepository.java

```diff
@@ -38,7 +38,10 @@
  */
 public class ProjectRepository {
 
-  private static final int PROJECT_LIST_PAGESIZE = 300;
+  // API may return fewer than 700 results in one request, but 600 and change is the
```

Comment:
I can't understand the second part of the comment clearly.

Reply:
`600 and change` is slang for "600-odd"

Reply:
Ah, thanks.